e2e/test/scenarios/visualizations-tabular/reproductions/18976-pivot-table-columns.cy.spec.js

```diff
@@ -30,4 +33,29 @@ describe("issue 18976", () => {
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.findByText("Showing 1 row");
   });
+
+  it("should not keep orphan columns rendered after switching from pivot to regular table (metabase#18817)", () => {
+    cy.createQuestion(
+      {
+        query: {
```

Comment:
Please add a limit (5?) to the query. No need to render all rows for this particular reproduction.

Reply:
added